jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Scanner.java

```diff
@@ -572,7 +614,7 @@ public void setReportExistingFilesOnStartup(boolean reportExisting)
         _reportExisting = reportExisting;
     }
 
-    public boolean getReportExistingFilesOnStartup()
+    public boolean isReportExistingFilesOnStartup()
```

Comment:
I guess if this method isn't referenced anywhere in jetty we might be able to get away with this method rename, but technically we should do a deprecation.

Reply:
This was changed to follow Jetty format / syntax.
Also, internally, the Scanner uses this method now, instead of the raw field.
This allows for folks extending Scanner have a better experience, as this now follows Java recommendations.